### PR TITLE
remove pagination form sigle pages

### PR DESCRIPTION
**Pull request by billyogat**
Fixes panr/hugo-theme-terminal#129

### CODE REVIEW

File: exampleSite/content/post/goodbye.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
++++
+title = "Goodby Friend"
```
Comment (panr) [inline exampleSite/content/post/goodbye.md:2]: Is this file necessary here?

Comment (billyogat) [reply exampleSite/content/post/goodbye.md:2]: Only added to confirm that the pagination was only working with posts. I can remove.

**Comment by panr** [comment]
Thanks! Merging it 🎉
